includes/admin/class-wc-rest-payments-vat-controller.php

```diff
@@ -66,7 +67,8 @@ public function register_routes() {
 	 */
 	public function validate_vat( $request ) {
 		$vat_number = $request->get_param( 'vat_number' );
-		return $this->forward_request( 'validate_vat', [ $vat_number ] );
+		$request    = Request::get( WC_Payments_API_Client::VAT_API, $vat_number );
+		return $request->send( 'wcpay_validate_vat_request' );
```

Comment:
Nitpick: Not too important for this short function but would use a new variable name to avoid confusion for other devs reading this again in the future: 


```suggestion
		$server_request    = Request::get( WC_Payments_API_Client::VAT_API, $vat_number );
		return $server_request->send( 'wcpay_validate_vat_request' );
```